modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/AbstractPageMemoryTableStorage.java

```diff
@@ -151,6 +152,12 @@ public void createIndex(String indexName) {
         throw new UnsupportedOperationException("Not implemented yet");
     }
 
+    @Override
+    @Nullable
+    public HashIndexStorage getHashIndex(int partitionId, String indexName) {
+        throw new UnsupportedOperationException("Not implemented yet");
+    }
+
```

Comment:
```suggestion
    @Override
    public @Nullable HashIndexStorage getHashIndex(int partitionId, String indexName) {
        throw new UnsupportedOperationException("Not implemented yet");
    }

```

Reply:
Do we have such formatting in the code style? If no, I would prefer to leave it as-is